### PR TITLE
make filefollow include user-defined timestamp format, and regex deli…

### PR DESCRIPTION
**Pull request by floren**
…miters

### CODE REVIEW

File: ingesters/fileFollow/main_linux.go [hunk 1]
```diff
@@ -199,6 +199,8 @@ func main() {
 			AssumeLocalTZ:           val.Assume_Local_Timezone,
 			IgnorePrefixes:          ignore,
 			TimestampFormatOverride: tsFmtOverride,
+			UserTimeRegex:           val.Timestamp_Regex,
```
Comment (traetox) [inline ingesters/fileFollow/main_linux.go:202]: any reason not to include this on the windows file too?

**Review by traetox — CHANGES_REQUESTED** [review]
add to windows file too

**Comment by floren** [comment]
ptal

**Review by traetox — APPROVED** [review]
LGTM

**Pull request merged**